### DISCUSSION TITLE
[API] how to get the process instance id when start a new process instance?

### DISCUSSION THREAD

**Discussion by ngsin** [apache/dolphinscheduler#9491] [Q&A]
how can i get the` process instance` id after start a new `process instance`?
`/dolphinscheduler/projects/{projectCode}/executors/start-process-instance`
i use this api to create, but it return just look like
`{"code":0,"msg":"success","data":null,"failed":false,"success":true}`

**Comment by Tandoy** [comment]
Maybe you can get process instance id according to the process name. Because the process name is not allowed to be repeated in a project.
